ntbs-integration-tests/HomePage/HomePageTests.cs

```diff
@@ -16,8 +16,8 @@ public class HomePageTests : TestRunnerBase
         [Fact]
         public async Task DismissAlert_CorrectlyDismissesAlertAndReturnsHomePage()
         {
-            using (var client = Factory.WithUser<NhsUserForAbingdonAndPermitted>()
-                .WithNotificationAndTbServiceConnected(Utilities.DRAFT_ID, Utilities.PERMITTED_SERVICE_CODE)
+            using (var client = Factory.WithUserAuth(TestUser.NhsUserForAbingdonAndPermitted)
+                .WithNotificationAndTbServiceConnected(Utilities.NOTIFIED_ID, Utilities.PERMITTED_SERVICE_CODE)

```

Comment:
Any reason this changed from draft to notified?

Reply:
Yes - the alert was for the Notified notification, it was previously only working by coincidence.